packages/gitmoji-changelog-core/src/index.js

```diff
@@ -57,8 +58,17 @@ function sanitizeVersion(version) {
   })
 }
 
+function groupCommitsByMessagesDistance(commits = []) {
+  const distanceGroups = groupSentencesByDistance(commits.map(commit => commit.message))
+
+  return distanceGroups.map(([first, ...siblings]) => ({
```

Comment:
first doesn't express its nature maybe we should rename it firstCommitIndex ? And the other one siblingsIndexes ?

Reply:
I did better to understand whats going on I think:
```js
  return distanceGroups
    .map(indexes => indexes.map(index => commits[index]))
    .map(([first, ...siblings]) => ({
      ...first,
      siblings,
    }))
```